meta-gnome/recipes-support/onboard/onboard_1.4.1.bb

```diff
@@ -10,7 +10,7 @@ SRC_URI = "https://launchpad.net/onboard/1.4/${PV}/+download/${BPN}-${PV}.tar.gz
 SRC_URI[md5sum] = "1a2fbe82e934f5b37841d17ff51e80e8"
 SRC_URI[sha256sum] = "01cae1ac5b1ef1ab985bd2d2d79ded6fc99ee04b1535cc1bb191e43a231a3865"
 
-inherit features_check setuptools3-base pkgconfig gtk-icon-cache mime-xdg
+inherit features_check distutils3 pkgconfig gtk-icon-cache gsettings mime-xdg
```

Comment:
@jpautler do you mind splitting the commit into two - one to revert Alex's change removing `gsettings` and another to use `distutils3`? Would help evaluate commits for upstreaming.

Reply:
Dropping since onboard was removed upstream.